config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -50,9 +50,9 @@ periodics:
       - ./kettle/monitor.py
       - --stale=6
       - --table
-      - k8s-gubernator:build.all
-      - k8s-gubernator:build.week
-      - k8s-gubernator:build.day
+      - k8s_infra_kettle:build.all
+      - k8s_infra_kettle:build.week
+      - k8s_infra_kettle:build.day
```

Comment:
does this need to move `cluster`s? or did we give access to the old cluster? (I think this is what arnaud was mentioning?)